ui/oopsyraidsy/data/06-ew/dungeon/lapis_manalis.ts

```diff
@@ -0,0 +1,64 @@
+import ZoneId from '../../../../../resources/zone_id';
+import { OopsyData } from '../../../../../types/data';
+import { OopsyTriggerSet } from '../../../../../types/oopsy';
+
+export type Data = OopsyData;
+
+const triggerSet: OopsyTriggerSet<Data> = {
+  zoneId: ZoneId.LapisManalis,
+  damageWarn: {
+    'Caladrius Transonic Blast': '7F17', // // front cone AoE, before boss 1
```

Comment:
There's no test for this like there is for raidboss, but can you add a common prefix to all of these ids, like `Lapis`?